doc/radosgw/config-ref.rst

```diff
@@ -7,6 +7,11 @@ The following settings may added to the Ceph configuration file (i.e., usually
 settings may contain default values. If you do not specify each setting in the
 Ceph configuration file, the default value will be set automatically.
 
+Configuration variables set under the ``[client.radosgw.{instance-name}]``
+section will not apply to rgw or radosgw-admin commands without an instance-name
+specified in the command. Thus variables meant to be applied to all RGW
+instances or all radosgw-admin commands can be put into the ``[global]`` or the
```

Comment:
@tchaikov I very quickly played around with this in ceph-ansible and I found that the settings in `[client.rgw.foo]` overrode those in `[client.rgw]` and `[client]`. The testing I did was not very rigorous by any means, but I think it's best to emphasize the vars should be in `[global]` if the intention is to have them apply everywhere.